bin/migration/azure_storage_utils/main.py

```diff
@@ -64,17 +64,20 @@ def update_durations_in_dataframe():
     df.to_csv("recordings.csv", index=False, mode="w", header=False)
 
 def update_recordings_table(conn):
-    df = pd.read_csv("recordings.csv", names=["Recording_ID", "Duration_seconds"])
-    for index, row in df.iterrows():
-        recording_id = row["Recording_ID"]
-        duration = row["Duration_seconds"]
-        update_query = f"UPDATE public.recordings SET duration = MAKE_INTERVAL(secs => {duration}) WHERE id = '{recording_id}'"
-        try:
-            cur = conn.connection.cursor()
-            cur.execute(update_query)
-            conn.connection.commit()
-        except Exception as e:
-            print(e)
+    df = pd.read_csv("recordings.csv")
```

Comment:
I think this line needs to go into the try/catch block or the error would not get trapped here, it will bubble up.

Reply:
That's done - good point thanks